src/Files.App/Views/Properties/GeneralPage.xaml

```diff
@@ -224,20 +236,23 @@
 							Text="{x:Bind ViewModel.UncompressedItemSize, Mode=OneWay}"
 							Visibility="{x:Bind ViewModel.IsUncompressedItemSizeVisibile, Mode=OneWay}" />
 
-						<MenuFlyoutSeparator
+						<!--  (Divider)  -->
+						<Border
 							Grid.Row="3"
 							Grid.Column="0"
 							Grid.ColumnSpan="2"
-							Margin="-12,0"
-							Background="{ThemeResource CardStrokeColorDefaultBrush}" />
+							Height="1"
+							Margin="-16,0"
+							Background="{ThemeResource DividerStrokeColorDefault}" />
```

Comment:
Can you share a screenshot of the before and after?

Reply:
![image](https://user-images.githubusercontent.com/62196528/232265910-a63b33f7-e338-4f14-9638-7bdd29c73864.png)

Reply:
This is a lot clearer, thank you.